Makefile

```diff
@@ -124,7 +124,7 @@ dev-race:
 
 # release builds and installs release binaries.
 release:
-	go install -tags='netgo' -ldflags='-s -w $(ldflags)' $(release-pkgs)
+	go install -tags='netgo dev' -ldflags='-s -w $(ldflags)' $(release-pkgs)
```

Comment:
Why the `dev` tag in the `release` target? 

Seems like there should be another `dev:` target?

Reply:
Oh, good catch! This is leftover debug code that I needed in order to run the service locally. Removed.